features/FEATURE_BLE/ble/services/HeartRateService.h

```diff
@@ -56,11 +56,24 @@ class HeartRateService {
     HeartRateService(BLE &_ble, uint8_t hrmCounter, uint8_t location) :
         ble(_ble),
         valueBytes(hrmCounter),
-        hrmRate(GattCharacteristic::UUID_HEART_RATE_MEASUREMENT_CHAR, valueBytes.getPointer(),
-                valueBytes.getNumValueBytes(), HeartRateValueBytes::MAX_VALUE_BYTES,
-                GattCharacteristic::BLE_GATT_CHAR_PROPERTIES_READ | GattCharacteristic::BLE_GATT_CHAR_PROPERTIES_NOTIFY),
         hrmLocation(GattCharacteristic::UUID_BODY_SENSOR_LOCATION_CHAR, &location),
-        controlPoint(GattCharacteristic::UUID_HEART_RATE_CONTROL_POINT_CHAR, &controlPointValue) {
+        controlPoint(GattCharacteristic::UUID_HEART_RATE_CONTROL_POINT_CHAR, &controlPointValue)
+    {
+        static uint8_t clientCharConfBytes[2] = {0, 0};  // clientCharacteristicConfiguration - length is 16 bits
+        static GattAttribute clientCharConf(BLE_UUID_DESCRIPTOR_CLIENT_CHAR_CONFIG, clientCharConfBytes, 2, 2, false);  // clientCharacteristicConfiguration
```

Comment:
static variables shouldn't be used, it prevent the instantiation of multiple, distinct heart rate services.